rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -100,11 +116,15 @@ def __str__(self) -> str:
         )
 
 
-class BaseAcquisitionsOrder(metaclass=ABCMeta):
-    _acquisitions: deque[AssetAcquisitionEvent]
+class BaseCostBasisMethod(metaclass=ABCMeta):
```

Comment:
Add a docstring to explain what this class is and/does. Better explain stuff in the docstring instead of the `#` comments